database/sql/V010__create_app_data.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE app_data (
+    app_data_hash bytea PRIMARY KEY,
+    app_code bytea,
+    referrer bytea,
+    file_blob bytea NOT NULL
```

Comment:
Any reason to not use `jsonp` as the field type?

Since this is a json data field... https://stackoverflow.com/a/65271803/1272513

If we ever want to query the data, this will be useful

Reply:
This will require formatting the string into json.RawMessage before inserting. Hence, the original string with all its empty spaces, etc would not be retrievable. This would deviate from the "treating it as ipfs file content". But maybe this is not soo bad. Let me investigate.

Reply:
I'm OK with this, as we should always be able to recover the original IPFS file from the `app_data_hash` right (given known CID version and multi-codec)?

Reply:
Hmm, yes good point.
If we do that then we are back at the deterministic serialization problem.
But also as we discussed, this might not be that big of a deal, if we have different hashed for same data as long as all versions properly tag the same user.

I'm in favor of the change.

cc @anxolin